engines/sci/graphics/video32.cpp

```diff
@@ -130,6 +130,18 @@ VideoPlayer::EventFlags VideoPlayer::playUntilEvent(const EventFlags flags, cons
 	_decoder->start();
 
 	EventFlags stopFlag = kEventFlagNone;
+
+	if (_subtitles.isLoaded()) {
+		const int16 h = g_system->getOverlayHeight(),
+			        w = g_system->getOverlayWidth();
+		_subtitles.setBBox(Common::Rect(20, h - 120, w - 20, h - 20));
```

Comment:
It has been mentioned before, but the subtitle position is hardcoded here. This needs to be calculated dynamically and tested with VMD videos from other games

Reply:
Thank you, I have added a calculation according to the draw rect of the video.
I think the position looks good on Phantasmagoria and GK2 for both full screen videos (phantasmagoria intro) and "windowed" videos, for both SDLSurface and OpenGL renderer.
Please note that it does draw outside of the window in the "windowed" case, which IMO is good for SDLSurface case because thus it avoids blocking the video with the opaque background.
although looking at the calculation itself, I'm not sure why